src/model.js

```diff
@@ -48,12 +60,15 @@ module.exports = function(koop) {
         if(req.url.includes('VectorTileServer')){
             layerId = "0";
             if(req.params.x && req.params.y && req.params.z){
-                const tileBBox = tileIndexToBBox(parseInt(req.params.x), parseInt(req.params.y), parseInt(req.params.z));
-                // logger.debug(`Z: ${req.params.z}, X: ${req.params.x}, Y: ${req.params.y}`);
-                // logger.debug(JSON.stringify(tileBBox));
+                const tileBBox = tileIndexToBBox(parseInt(req.params.x), parseInt(req.params.y), parseInt(req.params.z),
```

Comment:
This is hard to read, would also like to see this in its own function `getTileBBox` so that the details are hidden and we maintain abstraction level.

Reply:
This is already calling the tileIndexToBBox function.  Are you saying move the parseInt into the function? Or rename it getTileBBox and have it just take the req.params?

Reply:
Yes, right.

Reply:
Done